packages/members-api/lib/migrations/index.js

```diff
@@ -480,4 +480,23 @@ module.exports = class StripeMigrations {
             id: portalPlansSetting.id
         });
     }
+
+    async removeInvalidSubscriptions() {
+        const subscriptionModels = await this._StripeCustomerSubscription.findAll({
+            withRelated: ['stripePrice']
+        });
+        const invalidSubscriptions = subscriptionModels.filter((sub) => {
+            return !sub.toJSON().price;
+        });
+        if (invalidSubscriptions.length > 0) {
+            this._logging.warn(`Deleting ${invalidSubscriptions.length} invalid subscriptions`);
+            for (let index = 0; index < invalidSubscriptions.length; index++) {
```

Comment:
Minor but I think we prefer `for (subscription of invalidSubscriptions) {}` but nabd :)